android/festago/app/src/main/java/com/festago/festago/data/repository/FestivalDefaultRepository.kt

```diff
@@ -3,17 +3,19 @@ package com.festago.festago.data.repository
 import com.festago.festago.data.service.FestivalRetrofitService
 import com.festago.festago.domain.model.Festival
 import com.festago.festago.domain.repository.FestivalRepository
-import kotlinx.coroutines.delay
 
 class FestivalDefaultRepository(
     private val festivalRetrofitService: FestivalRetrofitService,
 ) : FestivalRepository {
     override suspend fun loadFestivals(): Result<List<Festival>> {
-        delay(1000)
-        val response = festivalRetrofitService.getFestivals()
-        if (response.isSuccessful && response.body() != null) {
-            return Result.success(response.body()!!.toDomain())
+        try {
+            val response = festivalRetrofitService.getFestivals()
+            if (response.isSuccessful && response.body() != null) {
+                return Result.success(response.body()!!.toDomain())
+            }
+            return Result.failure(Throwable("code: ${response.code()} message: ${response.message()}"))
+        } catch (e: Exception) {
+            return Result.failure(e)
```

Comment:
```suggestion
        return try {
            val response = festivalRetrofitService.getFestivals()
            if (response.isSuccessful && response.body() != null) {
                Result.success(response.body()!!.toDomain())
            }
            Result.failure(Throwable("code: ${response.code()} message: ${response.message()}"))
        } catch (e: Exception) {
            Result.failure(e)
        }
```

return 을 묶어주는게 어떨까요?